patches/chrome-browser-chrome_content_utility_manifest_overlay.json.patch

```diff
@@ -1,12 +0,0 @@
-diff --git a/chrome/browser/chrome_content_utility_manifest_overlay.json b/chrome/browser/chrome_content_utility_manifest_overlay.json
-index 03d7cd6b486001765096df3bfe95624fd2700c42..a5dbd36e97f36417b861c41a2bfb201a8640aba2 100644
---- a/chrome/browser/chrome_content_utility_manifest_overlay.json
-+++ b/chrome/browser/chrome_content_utility_manifest_overlay.json
-@@ -10,6 +10,7 @@
-           "payments.mojom.PaymentManifestParser",
-           "heap_profiling.mojom.ProfilingClient",
-           "proxy_resolver.mojom.ProxyResolverFactory",
-+          "tor.mojom.TorLauncher",
```

Comment:
should this be here?

Reply:
Nope, it's not a typo, we found this patch seems to be unnecessary since ads and rewards work fine with out patching here.

Reply:
ok sounds good